deps/chakrashim/chakracore.gyp

```diff
@@ -7,11 +7,10 @@
     'msvs_windows_target_platform_version_prop': '',
     'icu_args%': '',
     'icu_include_path%': '',
+    'icu_lib_path%': '',
     'linker_start_group%': '',
     'linker_end_group%': '',
     'chakra_libs_absolute%': '',
-    'icu_lib_path%': '',
-    'chakra_build_flags': '',
 
     # xplat (non-win32) only
     'chakra_config': 'Release',     # Debug, Release
```

Comment:
> Also, should we have Test build support for node-chakra?

We could, never tried. Probably a good feature if we can also get "jscript.config" working. Otherwise not very useful if we can't pass flags.